static/src/javascripts/projects/common/modules/experiments/tests/weekend-reading-promo.js

```diff
@@ -21,24 +27,34 @@ define([
         this.expiry = '2016-09-12';
         this.author = 'Kate Whalen';
         this.description = 'For just one pageview, show users a banner promoting the Weekend Reading email';
-        this.audience = 1;
+        this.audience = 0.5;
         this.audienceOffset = 0;
```

Comment:
We could put this at a higher offset so we don't overlap with the group that saw the snap the first time: https://github.com/guardian/frontend/pull/13704/files

that way your `hasSeenMessage()` condition below will just be to cover the odd edge case rather than exclude a sizeable chunk of our audience